projects/plugins/jetpack/_inc/client/at-a-glance/stats/index.jsx

```diff
@@ -154,30 +154,26 @@ export class DashStats extends Component {
 
 	dismissCard = () => {
 		this.setState( { emptyStatsDismissed: true } );
-		this.props.updateOptions( { dismiss_empty_stats_card: true } );
+		// this.props.updateOptions( { dismiss_empty_stats_card: true } );
```

Comment:
Potentially left over from testing? Commented out code here and on line 214.

Reply:
Good catch.
We should be good to remove that now.